spec/requests/v1/pacbio/requests_spec.rb

```diff
@@ -218,6 +218,83 @@
     end
   end
 
+  describe 'V1::Pacbio::Requests filtering', type: :request do
+    let!(:sample1) { create(:sample, name: 'SAMPLE-1') }
+    let!(:sample2) { create(:sample, name: 'SAMPLE-2') }
+
+    let!(:tube1) { create(:tube, barcode: 'TUBE-123') }
+    let!(:tube2) { create(:tube, barcode: 'TUBE-456') }
+
+    let(:request1) do
+      create(:pacbio_request).tap do |pr|
```

Comment:
You should use the factory to do this, The request factory already has support for adding a sample.

If you want to add a tube then use a new factory under request request_with_tube. You would then do 

`let(:request1) create(:pacbio_request_with_tube, sample:, tube:)`

This is more reusable. You can have a look at other factories for similar.